### PR TITLE
Families gamemode ported from /tg/, replacement for gangs!

### PR DESCRIPTION
**Pull request by Sinestia**
## About The Pull Request

This ports the families gamemode from /tg/ 
you can read about it here:
https://tgstation13.org/wiki/Families
https://github.com/tgstation/tgstation/pull/49274
(albeit, the information in those is mildly outdated. For example the space police arrive faster if death is occurring.)

In essence, it's a gangs replacement that features
A. a little more of an RP bent
B. 3 teams. 
C. Voluntary, and there is no gamer gear as a reward. You do it for the fun of being part of an organized greytide.  

This seems to work in my private, one-person testing. My changes to make it work hopefully aren't too terrible. 

Bee might want to redo the flavor text and some of the assets. I avoided doing so, my goal was to get it in a working state first to see if people enjoy it. 
## Why It's Good For The Game

Players like to ask for gangs to be forced/brought back

That's not going to happen but maybe this more RP-oriented, more modern version will be fun, as well as add more diversity by having a roundtype that isn't "station vs bad men" or "full greenshift."
## Changelog
:cl:Froststahr, credit to the original authors!
add: Families gamemode has been ported. Organized crime has infiltrated NT stations! 
del: The original gangs gamemode has been removed, due to occupying a lot of the uhhh namespace? 
sprite: Families sprites, sprays for all of the gangs, icons, and some outfit sprites. 
sound: Families related sounds, such as the roundstart theme
/:cl:

### CODE REVIEW

**Comment by eeSPee** [comment]
I personally hate families for 5 reasons:
* Families are a super weak version gangster. You don't get any perks or proper way to "coordinate" as a family. All you get is a ugly jumpsuit and that's it. The thing is, they don't have enough good equipment or quirks to justify them being a proper 'antagonist'.
* It's very unsure to tell if you are a proper antagonist or not, what you are allowed to do, and not. Are you allowed to attack sec? Are you allowed to destroy station? What if sec gets one of your members? What if other family attacks you? Who is other family? How can you tell them apart? Gang suits are super generic colored jumpsuits (that get hidden by your overalls anyway) so it's nigh impossible to tell who is other, or even your family.
* Despite the above reasons, security is still allowed to valid hunt you and treat you as a proper antagonist. 
* The voluntary conversion thing. Yea, it didn't work on devils, and it works even less here, for the reasons detailed above. You wouldn't wanna convert into a family so you turn into an antag "in name only" that gets validhunted by sec.
* Another problem was that the objectives were weird and poorly communicated. On families, you can spend the entire shift as a 'family member' without getting your objective or anyone caring about telling you what it is. Can't blame them though; objectives are either "make sure all members die in a coffin", or "have a lot of booze" or whatever. These were really pointless and most of the time accomplished by doing nothing.

TL;DR families is a combination of devils and gangs; it brings the negatives of both, and the positives of neither. It's a horrible game mode. Please don't port it to bee.

**Comment by nikk735** [comment]
test merge to see how bad it is?

**Comment by Archanial** [comment]
Families are trash

**Comment by Sinestia** [comment]
> 
> 
> I personally hate families for 5 reasons:
> 
>     * Families are a super weak version gangster. You don't get any perks or proper way to "coordinate" as a family. All you get is a ugly jumpsuit and that's it. The thing is, they don't have enough good equipment or quirks to justify them being a proper 'antagonist'.
> 
>     * It's very unsure to tell if you are a proper antagonist or not, what you are allowed to do, and not. Are you allowed to attack sec? Are you allowed to destroy station? What if sec gets one of your members? What if other family attacks you? Who is other family? How can you tell them apart? Gang suits are super generic colored jumpsuits (that get hidden by your overalls anyway) so it's nigh impossible to tell who is other, or even your family.
> 
>     * Despite the above reasons, security is still allowed to valid hunt you and treat you as a proper antagonist.
> 
>     * The voluntary conversion thing. Yea, it didn't work on devils, and it works even less here, for the reasons detailed above. You wouldn't wanna convert into a family so you turn into an antag "in name only" that gets validhunted by sec.
> 
>     * Another problem was that the objectives were weird and poorly communicated. On families, you can spend the entire shift as a 'family member' without getting your objective or anyone caring about telling you what it is. Can't blame them though; objectives are either "make sure all members die in a coffin", or "have a lot of booze" or whatever. These were really pointless and most of the time accomplished by doing nothing.
> 
> 
> TL;DR families is a combination of devils and gangs; it brings the negatives of both, and the positives of neither. It's a horrible game mode. Please don't port it to bee.

They're a different kind of antag in that the dynamic is them vs the other groups of the same, within the station. That's their impact on the round, yes, it's different from the usual sort of antags and is more RP-based, but that's ok and it's something I think is very much worth emphasizing about its "place." Especially when we do have extended. 

 Also, I think the devil comparison is unfair. The problem with devil is that the incentives driving player behavior are absolutely wrong. You join a gang because you want to directly take part in the mode. In devil you seek out the devil for pure powergaming purposes or so you can kill the valid man. 

 A lot of the rules are much more clear than you make them out to be. It's made clear that you can't destroy the station, security are not allowed to validhunt you as an EOC (they have to witness you committing crimes) and you are allowed to combat other families with lethal force. 



 I agree that the objectives aren't very well laid out, they ultimately don't matter either design-wise however. I believe the latest revision of the gamemode actually gets rid of them (but not the version I have.) I might have to grab that. Along with a lot of the new sprites which look vastly superior, and icon indicators to tell who's with who.

**Comment by ghost** [comment]
> Families are trash

That's not criticism.

**Comment by ike709** [comment]
Pretty sure there's like a dozen follow-up PRs.

**Comment by Sinestia** [comment]
> Pretty sure there's like a dozen follow-up PRs.

My basis was the latest version, so they should be in unless I missed some sprite changes.

**Comment by PowerfulBacon** [comment]
Voluntary and no gamer gear sound like they would make pretty poor antags. The last gamemodes that attempted voluntary conversion were awful and hated. Upon calling them put they will get validhunts almost straight away when someone says no to conversion.

Looking at the original PR makes me even more weary of it, it has way too many similarities to other failed gamemodes and admin events.
 - Voluntary sign up
 - No way to identify if someone is really in your gang. Unless you wear clothes
 - Wearing clothes will get you validhunted by sec
 - Voluntary but absolutely no incentive to join apart from you can do some minor grief
 - No gear to deal with sec.
 - Requires RP to work rather than game mechanics and design.
 - Also a large portion looks like it requires admin intervention (The space cops ERT part)
 
Although I think its gunna be bad, you never really know so it's still worth a test I guess.

**Comment by eeSPee** [comment]
> They're a different kind of antag in that the dynamic is them vs the other groups of the same, within the station. That's their impact on the round, yes, it's different from the usual sort of antags and is more RP-based, but that's ok and it's something I think is very much worth emphasizing about its "place." Especially when we do have extended.

Yes but here is where other problems arise; you're not allowed to randomly attack people, and you cannot tell who is part of an enemy gang. Unless someone overhears you/sees you being part of an enemy gang and you get validhunted and killed by random people.
 
> Also, I think the devil comparison is unfair. The problem with devil is that the incentives driving player behavior are absolutely wrong. You join a gang because you want to directly take part in the mode. In devil you seek out the devil for pure powergaming purposes or so you can kill the valid man.

That's the problem. You get nothing from joining a gang... And participating in it offers you absolutely nothing. So nobody will bother with them.
 
> A lot of the rules are much more clear than you make them out to be. It's made clear that you can't destroy the station, security are not allowed to validhunt you as an EOC (they have to witness you committing crimes) and you are allowed to combat other families with lethal force.

So then, what's the point of a family game mode?

> I agree that the objectives aren't very well laid out, they ultimately don't matter either design-wise however. I believe the latest revision of the gamemode actually gets rid of them (but not the version I have.) I might have to grab that. Along with a lot of the new sprites which look vastly superior, and icon indicators to tell who's with who.

This makes families even more boring and pointless!



I've played family game mode, and it's a poorly thought out mess of an antag. Most of the time, nobody bothers with them (not even gang leaders). The most effort they make is littering the entire station with those useless books nobody bothers with. For the rest of the station, it's a regular greenshift.

If you do decide to become a family, best case scenario, nobody bothers to give you your goal, and you have no way of telling your other family members, so nothing changes, Usually, one family snowballs only to valid hunt other families. This family gathers all the shitters and gangs up on anyone who they have confirmed to be other family. They also use the family so they are allowed to tide and stand up to security. It's not fun.

Oh yea, and when the 1h mark hits, space cops come and validhunt all gangsters, anyway.

Families are not an antag, so they shouldn't have an entire shift dedicated to them.

**Comment by Penwin0** [comment]
> * No way to identify if someone is really in your gang. Unless you wear clothes

Im like 99% sure they get huds that show it much like revs cult or gang

**Comment by Sinestia** [comment]
> 
> 
> > * No way to identify if someone is really in your gang. Unless you wear clothes
> 
> Im like 99% sure they get huds that show it much like revs cult or gang

They don't, not without some of the non-merged revisions that were made.




 So, not that I don't have responses to what was said. But I think it may be more useful to implement some changes. 
 Otherwise it'll probably just be banal back and forths, and some concerns definitely can be addressed even if I disagree with others (e.g. I think there is some inherent value behind tribalism.)

**Comment by Effemu** [comment]
>Players like to ask for gangs to be forced/brought back

i can guarantee they dont ask for gangs to have a roleplay round

also families suck
sage is lrp

**Comment by Archanial** [comment]
Also this mode just spawns space cops who just go murderbone and end the round at 1 hour mark.

**Comment by ghost** [comment]
> > Players like to ask for gangs to be forced/brought back
> 
> i can guarantee they dont ask for gangs to have a roleplay round
> 
> also families suck
> sage is lrp

You know that /tg/ has 3 LRP servers aswell?

It's insane how people think talking to someone = RP.

**Comment by Effemu** [comment]
> 
> 
> > > Players like to ask for gangs to be forced/brought back
> > 
> > 
> > i can guarantee they dont ask for gangs to have a roleplay round
> > also families suck
> > sage is lrp
> 
> You know that /tg/ has 3 LRP servers aswell?
> 
> It's insane how people think talking to someone = RP.

you know that Sage doesnt have people who talk?
its insane how people think beestation playerbase is capable of saying anything but "he valid"

anyways devil was removed for many reasons, one of which was 0 interaction between players apart from caving in antag's skull with a toolbox
So porting families which is purely roleplay focused gamemode is a great idea if you want rounds lasting ~10 minutes

**Comment by BrotherHangyul** [comment]
There better be a fucking yakuza

**Comment by wraith-54321** [comment]
maybe some kind of cult type chat with a cooldown/only leaders can send messages

**Comment by Sinestia** [comment]
I'm gonna close this

 If I reintroduce it/something like it it's probably going to be extremely different. 

 Basic bullet list of what I think needs to be introduced:

- Some gear should return, og gangs gear was fucked up so not to that degree tho
- Conversion will still be voluntarily, you gotta bear wit me tho
- Tightly limited gang sizes, instead of only keeping them within X size of eachother. Think 5 man gangs.
- Basebuilding incentives and visual indicators for marking territory other than a normal spray alone that blends in. 
- No space cops, except maybe as a custom ERT if really wanted. 
 
Basically making it more similar with goon and taking into account the feedback I've been able to find across the net to it and families. 

If I actually put some effort into this I'll make a forum post for feedback and resubmit as a draft
